docs/upgrade-guide.md

```diff
@@ -4,6 +4,18 @@ ___You must make backups before attempting an upgrade.___
 
 This guide explains how to upgrade a Percona XtraDB Cluster to version 8.4 without causing downtime. This process is called a “rolling upgrade,” which means you can upgrade the cluster one node at a time without shutting down the whole cluster. Keep in mind that rolling upgrades to 8.4 are only supported if your current version is 8.0 or newer. Be sure you are running on the latest 8.0 version before you upgrade to 8.4.
 
+!!! warning
+
+    Upgrade to the newest 8.0 version, but ensure it is not newer than the corresponding 8.4 version you plan to upgrade to. 
+    
+    This precaution is necessary due to potential Galera Communication System (GCS) protocol upgrades. If nodes in the cluster use different GCS protocol versions, it can lead to compatibility issues, potentially causing disruptions or preventing nodes from joining the cluster. 
```

Comment:
I would simply say: If joining node uses lower protocol version than the existing cluster, the cluster will prevent this node to join the cluster.